src/App.tsx

```diff
@@ -1,39 +1,75 @@
 import React from 'react';
+import { Clock } from './Clock/Clock';
 import './App.scss';
 
-function getRandomName(): string {
-  const value = Date.now().toString().slice(-4);
-
-  return `Clock-${value}`;
+interface State {
+  clockName: string;
+  currentTime: string;
+  hasClock: boolean;
 }
 
-export const App: React.FC = () => {
-  const today = new Date();
-  let clockName = 'Clock-0';
+export class App extends React.Component {
+  state: State = {
+    clockName: 'Clock-0',
+    currentTime: new Date().toUTCString().slice(-12, -4),
+    hasClock: true,
+  };
+
+  componentDidMount(): void {
```

Comment:
It's extra, TS already knows about React types 
```suggestion
  componentDidMount() {
```